### PR TITLE
Docker : building + publishing

### PR DESCRIPTION
**Pull request by Chavithra**
# Description

**BUILD**
Putting the building part into a `build.sh` script.
The `docker-compose.yml` file doesn't allow setting the order in which the images are built.
The `depends_on` is meant to order the start not the build.

**PUBLISHING**
Making sure the publishing on fork working fine.
So we can test the package launching this Github Action in a fork.

**INTEGRATION TESTS**
Also moving some documentation + integration test script into the right folder.

**DOCUMENTATION**
Updated the documentation related to the installation.

### CODE REVIEW

**Comment by Chavithra** [comment]
@simmonsj330 is it possible to check this `./openbb_terminal/miscellaneous/scripts/README.md` ?

File: openbb_terminal/helper_funcs.py [hunk 1]
```diff
@@ -1149,7 +1149,6 @@ def replace_user_timezone(user_tz: str) -> None:
     """
     if is_timezone_valid(user_tz):
         dotenv.set_key(USER_ENV_FILE, "OPENBB_TIMEZONE", user_tz)
-        os.environ["OPENBB_TIMEZONE"] = user_tz
```
Comment (colin99d) [inline openbb_terminal/helper_funcs.py:1152]: Is there a reason this PR removes my previous bug fix? The bug of TZ not getting updated right away is reintroduced with this change.
Screen Shot 2022-10-19 at 7 10 56 AM

**Review by colin99d — CHANGES_REQUESTED** [review]

File: openbb_terminal/helper_funcs.py (hunk 1 above)
Comment (Chavithra) [reply openbb_terminal/helper_funcs.py:1152]: I have seen that fixed it in local, didn't push it yet

File: openbb_terminal/DOCKER_README.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+
+```bash
+IP=$(ifconfig | grep inet | grep -v -e "127.0.0.1" | awk '$1=="inet" {print $2}')
+xhost + $IP
+```
+
+Now we can run the docker container, adding the display to the environment:
+
+```bach
+docker run -it --rm --env-file=path/to/setenv --env DISPLAY=$IP:0 ghcr.io/openbb-finance/openbbterminal/poetry:X.Y.Z
```
Comment (colin99d) [inline openbb_terminal/DOCKER_README.md:154]: I think `--env-file=path/to/setenv` needs to be replaced with a volumes solution

File: openbb_terminal/helper_funcs.py (hunk 1 above)
Comment (colin99d) [reply openbb_terminal/helper_funcs.py:1152]: Are we going to undo it?

File: README.md [hunk 3]
```diff
@@ -66,44 +66,48 @@
...
-As a modern Python-based environment, OpenBBTerminal opens access to numerous Python data libraries in Data Science
+As a Python-based environment, OpenBBTerminal opens access to numerous Python data libraries in Data Science
 (Pandas, Numpy, Scipy, Jupyter), Machine Learning (Pytorch, Sklearn, Flair), and Data Acquisition
 (Beautiful Soup, and numerous third-party APIs).
 
-## Installation
+## 2. Installation
 
-If you wish to install the Terminal, there are currently four options:
+If you wish to install the OpenBBTerminal, there are currently four options:
```
[the remark this answers is not in the document]
Comment (Chavithra) [reply README.md:97]: I have added the Pypi.org method.

File: openbb_terminal/DOCKER_README.md (hunk 2 above)
Comment (Chavithra) [reply openbb_terminal/DOCKER_README.md]: You are right.

**Pull request merged**